packages/core-data/src/entities.js

```diff
@@ -39,6 +39,7 @@ export const rootEntitiesConfig = [
 				'url',
 			].join( ',' ),
 		},
+		plural: '__unstableBases',
```

Comment:
Do we need this for BC? because this particular entity doesn't support multiple records.

Reply:
Yes. This is for BC. I mentioned this in the PR description.

Reply:
Ooops, sorry :P

Reply:
I guess it could use an extra comment that explains why we need it then.